### PR TITLE
Fix required rule for updating unchanged attachment field

### PR DESCRIPTION
**Pull request by wimski**
Making a paperclip field required resulted in an issue that when updating the resource without changing the paperclip field value, it return a validation error. This PR fixes that by checking if the underlying model has an attachment and if so removing the required rule from the update rules.

### CODE REVIEW

File: phpstan.neon.dist [hunk 1]
```diff
@@ -0,0 +1,11 @@
+includes:
+    - vendor/nunomaduro/larastan/extension.neon
+    - vendor/phpstan/phpstan-mockery/extension.neon
+    - vendor/phpstan/phpstan-phpunit/extension.neon
+    - vendor/phpstan/phpstan-phpunit/rules.neon
```
Comment (daniel-de-wit) [inline phpstan.neon.dist:5]:
```suggestion
    - vendor/phpstan/phpstan-phpunit/rules.neon
    - vendor/thecodingmachine/phpstan-safe-rule/phpstan-safe-rule.neon
```

**Review by daniel-de-wit — APPROVED** [review]
Looks good to me, thanks!

**Comment by wimski** [comment]
Even-though the checks now pass, I'm still having doubts about the dependencies. I'm having a hard time determining what the lowest versions are we can support.

**Comment by wimski** [comment]
# 😭 
https://github.com/laravel/nova-issues/issues/3388

**Pull request merged**